dChatServer/ChatServer.cpp

```diff
@@ -33,21 +33,29 @@ namespace Game {
 	dConfig* config = nullptr;
 	dChatFilter* chatFilter = nullptr;
 	AssetManager* assetManager = nullptr;
-	bool shouldShutdown = false;
+	Game::signal_t lastSignal = 0;
 	std::mt19937 randomEngine;
 	PlayerContainer playerContainer;
 }
 
 Logger* SetupLogger();
 void HandlePacket(Packet* packet);
 
+void OnSignal(int signal)
+{
```

Comment:
This block of code is repeated, would this be able to be moved to Servers.cpp or does each server require a definition like this?

Reply:
Yes